xarray/tests/test_backends.py

```diff
@@ -1126,13 +1128,11 @@ def _create_cf_dataset():
 
     def test_grid_mapping_and_bounds_are_not_coordinates_in_file(self) -> None:
```

Comment:
Some of these CF convention handling test really only need to be run with one backend.

cc @kmuehlbauer in case that interests you.  Similarly with the `"coordinates"` roundtripping